### ISSUE TITLE
Reset app state when running a test

### ISSUE THREAD

**Issue by BenedictP** [mobile-dev-inc/maestro#34]
When I run a test, the app state should be reset. Otherwise, the tests won't be deterministic (for my use case).
For example: After a fresh install, we show a GDPR screen and after the user accepts it, it won't be shown again.
It's mainly an issue while I am writing tests locally. I always need to reset the app by hand.

**Comment by dmitry-zaitsev** [comment]
Yeah, that makes sense. At mobile.dev, we usually run `adb shell pm clear {package name}`, but I can see this being handy for CLI in general

**Comment by dmitry-zaitsev** [comment]
This is going to be addressed in #41

**Comment by dmitry-zaitsev** [comment]
@BenedictP  This is now available in version 1.1.0

Here is how to use it:
```
- launchApp:
    appId: "com.example.app"
    clearState: true
```
